client/my-sites/subscribers/queries/use-subscribers-query.tsx

```diff
@@ -22,7 +23,13 @@ const useSubscribersQuery = ( {
 	sortTerm = SubscribersSortBy.DateSubscribed,
 	filterOption = SubscribersFilterBy.All,
 }: SubscriberQueryParams ) => {
-	const hasManySubscribers = useManySubsSite( siteId );
+	const { hasManySubscribers, isLoading } = useManySubsSite( siteId );
+	const [ shouldFetch, setShouldFetch ] = useState( false );
+	useEffect( () => {
```

Comment:
Do you need useEffect here? Can't you just do

```
const shouldFetch = ! isLoading;
```

Or use isLoading directly in the enabled call?

Reply:
Good point! It does the job 👍